lib/aws-record/record/item_operations.rb

```diff
@@ -95,26 +112,58 @@ def assign_attributes(opts)
       end
 
       # Mass assigns the attributes to the model and then performs a save
+      #
+      # You can use the +:force+ option to perform a simple put/overwrite
+      # without conditional validation or update logic.
+      #
+      #
+      # @example Usage Example
+      #   class MyModel
+      #     include Aws::Record
+      #     integer_attr :id,   hash_key: true
+      #     string_attr  :name, range_key: true
+      #   end
+      #
+      #   model = MyModel.new(id: 4, name: "John")
+      #   model.name  # => "John"
+      #   model.save   
+      # 
+      #   model.update(name: "Janeß")
+      #   model.name       # => "Jane"
+      #   model.persisted? # => true
```

Comment:
I think we're thinking of the `dirty?` method here, which would be false since the attribute changes and save were concurrent. This also would benefit from an example where we are not changing the key.